quiz.py

```diff
@@ -0,0 +1,144 @@
+import random
+from abc import ABCMeta, abstractmethod
+from sqlalchemy.sql.expression import exists
+from sqlalchemy.sql.functions import func
+from models.painter import Painter
+from models.painting import Painting
+from models.saint import Saint
+
+
+class MultipleChoiceQuestion:
+    def __init__(self, question, correct_answer, option_list):
+        self.question_dict = question
+        self.question = question.values()[0]
+        self.correct_answer = correct_answer
+        self.option_list = option_list
+
+
+class Quiz:
+    __metaclass__ = ABCMeta
+
+    def __init__(self):
+        self.number_of_rounds = 0
+        self.score = []
+
+        # current
+        self.current_question = None
+
+        self.right_guesses = sum(self.score)
+        self.wrong_guesses = len(self.score) - self.right_guesses
```

Comment:
this initialization of `right/wrong_guesses` is actually useless, because the `score` is always empty in the beginning